operator/config/samples/python_ execution/taskdefinition_pyfunction_configmap.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: lifecycle.keptn.sh/v1alpha2
```

Comment:
let's remove the whitespace from name of the folder containing these files (`python_ execution`)